src/outputs/outputs.hpp

```diff
@@ -64,12 +64,13 @@ struct OutputParameters {
   bool single_precision_output;
   bool sparse_seed_nans;
   int hdf5_compression_level;
+  bool write_xdmf;
   // TODO(felker): some of the parameters in this class are not initialized in constructor
   OutputParameters()
       : block_number(0), next_time(0.0), dt(-1.0), file_number(0),
         include_ghost_zones(false), cartesian_vector(false),
         single_precision_output(false), sparse_seed_nans(false),
-        hdf5_compression_level(5) {}
+        hdf5_compression_level(5), write_xdmf(true) {}
```

Comment:
Any specific reason to use `true`as default?
hst, histogram, Ascent all do not need/support xdmf.

Reply:
backwards compatibility only. If we're fine breaking that, I'm fine making this default to false.

Reply:
Ah, I see what you mean. I've set the default initialized value to `false`, but the default value from `GetOrAddBoolean` when called by `ENABLE_HDF5` to `true`.